compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/syntax/tree/CompoundAssignmentOperator.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerinalang.compiler.syntax.tree;
+
+import io.ballerinalang.compiler.internal.parser.tree.STNode;
+
+public class CompoundAssignmentOperator extends Statement {
```

Comment:
Do we need this node? Can the two fields `private Token binaryOperator` and `private Token equalsToken`  be embedded to the `CompoundAssignmentStatement` node itself?

Reply:
I put it as a node, since in the grammar it is specified separately

Reply:
yea, but we can merge the two right, since CompoundAssignmentOperator is not re-used?

Reply:
alright, I ll merge the two :)